diff -aurN blender-2.66.orig/source/blender/blenfont/intern/blf_lang.c blender-2.66/source/blender/blenfont/intern/blf_lang.c
--- blender-2.66.orig/source/blender/blenfont/intern/blf_lang.c	2013-02-26 23:47:21.745908683 +0100
+++ blender-2.66/source/blender/blenfont/intern/blf_lang.c	2013-02-26 23:49:24.275246886 +0100
@@ -78,7 +78,7 @@
 
 static void fill_locales(void)
 {
-	char *languages_path = BLI_get_folder(BLENDER_DATAFILES, "locale");
+	char *languages_path = BLI_get_folder(BLENDER_SYSTEM_LOCALE, NULL);
 	LinkNode *lines = NULL, *line;
 	char *str;
 	int idx = 0;
@@ -185,7 +185,7 @@
 void BLF_lang_init(void)
 {
 #ifdef WITH_INTERNATIONAL
-	char *messagepath = BLI_get_folder(BLENDER_DATAFILES, "locale");
+	char *messagepath = BLI_get_folder(BLENDER_SYSTEM_LOCALE, NULL);
 
 	if (messagepath) {
 		bl_locale_init(messagepath, TEXT_DOMAIN_NAME);
diff -aurN blender-2.66.orig/source/blender/blenlib/BLI_path_util.h blender-2.66/source/blender/blenlib/BLI_path_util.h
--- blender-2.66.orig/source/blender/blenlib/BLI_path_util.h	2013-02-26 23:47:21.738908435 +0100
+++ blender-2.66/source/blender/blenlib/BLI_path_util.h	2013-02-26 23:47:36.524432165 +0100
@@ -60,6 +60,7 @@
 #define BLENDER_SYSTEM_DATAFILES    52
 #define BLENDER_SYSTEM_SCRIPTS      53
 #define BLENDER_SYSTEM_PYTHON       54
+#define BLENDER_SYSTEM_LOCALE       55
 
 /* for BLI_get_folder_version only */
 #define BLENDER_RESOURCE_PATH_USER      0
diff -aurN blender-2.66.orig/source/blender/blenlib/intern/path_util.c blender-2.66/source/blender/blenlib/intern/path_util.c
--- blender-2.66.orig/source/blender/blenlib/intern/path_util.c	2013-02-26 23:47:21.738908435 +0100
+++ blender-2.66/source/blender/blenlib/intern/path_util.c	2013-02-26 23:47:36.524432165 +0100
@@ -1098,6 +1098,9 @@
 			if (get_path_local(path, "python", subfolder, ver)) break;
 			if (get_path_system(path, "python", subfolder, "BLENDER_SYSTEM_PYTHON", ver)) break;
 			return NULL;
+
+		case BLENDER_SYSTEM_LOCALE:
+			BLI_strncpy(path, "/usr/share/locale", FILE_MAX); break;
 	}
 	
 	return path;
diff -aurN blender-2.66.orig/source/creator/CMakeLists.txt blender-2.66/source/creator/CMakeLists.txt
--- blender-2.66.orig/source/creator/CMakeLists.txt	2013-02-26 23:47:21.731908187 +0100
+++ blender-2.66/source/creator/CMakeLists.txt	2013-02-26 23:47:36.525432200 +0100
@@ -294,9 +294,9 @@
 if(WITH_INTERNATIONAL)
 	install(
 		DIRECTORY
-			${CMAKE_SOURCE_DIR}/release/datafiles/locale
+			${CMAKE_SOURCE_DIR}/release/datafiles/locale/
 			${CMAKE_SOURCE_DIR}/release/datafiles/fonts
-		DESTINATION ${TARGETDIR_VER}/datafiles
+			DESTINATION ${CMAKE_INSTALL_PREFIX}/share/locale
 		PATTERN ".svn" EXCLUDE
 	)
 endif()
