diff -aurN blender-2.66.orig/source/creator/CMakeLists.txt blender-2.66/source/creator/CMakeLists.txt
--- blender-2.66.orig/source/creator/CMakeLists.txt	2013-02-25 12:37:26.379546996 +0100
+++ blender-2.66/source/creator/CMakeLists.txt	2013-02-25 13:09:03.624613970 +0100
@@ -242,7 +242,7 @@
 		if(WITH_PYTHON_MODULE)
 			set(TARGETDIR_VER ${PYTHON_LIBPATH}/python${PYTHON_VERSION}/site-packages/${BLENDER_VERSION})
 		else()
-			set(TARGETDIR_VER ${CMAKE_INSTALL_PREFIX}/share/blender/${BLENDER_VERSION})
+			set(TARGETDIR_VER ${CMAKE_INSTALL_PREFIX}/share/blender)
 		endif()
 	endif()
 
